homeassistant/components/mqtt/config_flow.py

```diff
@@ -205,7 +208,9 @@ def update_password_from_user_input(
 class FlowHandler(ConfigFlow, domain=DOMAIN):
     """Handle a config flow."""
 
-    VERSION = 1
+    # Can be bumped to version 2.1 with HA Core 2026.1.0
+    VERSION = ENTRY_VERSION  # 1
+    SUBVERSION = ENTRY_MINOR_VERSION  # 2
```

Comment:
This is not correct, it should be MINOR_VERSION. Please check in tests the config entry is created with version 1.2

Reply:
corrected